### DISCUSSION TITLE
Regulate community claims to make them more reasonable and fair.

### DISCUSSION THREAD

**Discussion by woshidama323** [filecoin-project/notary-governance#933] [General]
### English
Community claims serve as an effective tool for pinpointing issues and facilitating prompt corrections, thus ensuring the FILPlus project continues to operate in an orderly and fair manner. This process is extremely beneficial.

But at the same time We also are concerned that if someone raises dubious queries about some LDNs and cannot receive a definitive conclusion for an extended period, it impacts all LDNs associated with the SPs connected to this particular LDN. With just a piece of insufficient evidence, they can easily cast doubt on an LDN, affecting all the clients, SPs, and notaries involved with the LDN. 

Allow me to reference #913 as an example. The Hidde team raised concerns about Topblocks, which were not acknowledged by Topblocks. Conversely, the evidence provided by Topblocks was not accepted by the Hidde team either, resulting in a seemingly endless cycle of disagreement. Consequently, all projects associated with Topblocks, linked to #913, are impacted. This includes projects such as [#1248](), #1974, [#2021](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/2021), and more.

If the Filplus team is unable to deliver a decision promptly, this issue will continue to linger. **The Hidde team would take this as a factual point and disseminate it widely within the community**, which is highly unjust towards us, Topblocks. As a result, our collaborative plans with other teams, our individual sealing plans, along with our commercial reputation are all impacted.

![image](https://github.com/filecoin-project/notary-governance/assets/11014169/465571a0-0334-4920-b8dc-ca9d32bb578b)

It's been nearly two weeks since #913, and many of our partners have expressed concern. We have explicitly highlighted our concerns, as can be seen in the response to #913. However, the team that raised this issue shows no concern, nor do they seem to bear any responsibility. This situation is patently unfair, leaving us powerless. We earnestly hope that the FILPlus team can expedite your decision.

![image](https://github.com/filecoin-project/notary-governance/assets/11014169/ea7eb007-f598-469c-8456-afed34e0f3fb)

Therefore, we propose that the FILPlus team should provide clear standards, regulations, and a reasonable timeline for decision-making for such issues. We suggest incorporating these in [this proposal](https://github.com/filecoin-project/notary-governance/issues/922) to mitigate the potential impact on clients, SPs, and notaries during the process.

Consider the scenario where issues such as #913 (alongside similar ones like #914, #915, #916, #917, etc.) do not garner the necessary attention from FILPlus. In such a case, we might encounter even more analogous incidents in the future. How are SPs supposed to put into action plans intended to expedite the growth of network computing power with the ecosystem, as well as initiatives aimed at fostering a thriving ecosystem?

### 中文
社区主张能够很好的发现问题，及时纠正，让FILPlus项目能够有秩序的公平的运行下去，这非常有帮助
但同时我们也担忧如果有人恶意给出一些LDN的质疑，涉及到这个LDN的所有SP 关联client 的所有LDN都受到影响，
因为社区主张的发起者给出一个不充分的证据，就可以轻易的质疑一个LDN，造成涉及到LDN中所有的client SP 公证人的影响，

请允许我拿#913作为案例，hidde团队质疑topblocks的理由，Topblocks不认可，Topblocks给出的证据，hidde团队不认可，那么这样就陷入无意义的循环当中，
这样因为#913所关联到的topblocks的所有项目都受到影响，比如 #1248 #1974 #2021等等
而如果filplus团队不能及时给予裁决，那么这个问题会一直存在，而hidde团队以此当作事实在社区当中广泛传播（对此我们完全有理由要求hidde团队向Topblocks团队道歉），
对我们topblocks非常不公平，我们跟其他团队的合作计划，我们自己的封装计划，以及商业信誉都受到影响 

#913 已经过去快两周，已经有很多合作伙伴对我们有担忧，我们明确提出我们的担忧，请看#913的回复，发起issue的团队对此并不关心，也不需要负责，这本身非常不公平，但我们什么都做不了，期望FILPlus团队能尽快给予裁决

所以我们建议FILPlus 团队能够对这类issue给出非常明确的标准，规定，以及合理的裁决时间，并加入到#922中，降低处理过程中对client/SP以及公证人的潜在影响。
设想下如果#913 （#914，#915，#916，#917等类似的问题）等问题没有得到FILPlus的重视，那么后面还会有更多的类似事件，SP希望与生态一起快速增加全网算力的计划，繁荣生态的计划如何实施？

**Comment by ghost** [comment]
Hi- Normally, an issue will stay open for two week period and if there is consensus, it is reviewed on governance call and closed. 

@woshidama323 - It looks as though 913 is still active with comments and inputs from community members, including T&T WG acknowledgement of this fact.  It seems there is more to understand, it's not clear.

**Reply by lyjmry** [reply]
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/2008#issuecomment-1643645187

**Comment by max1015070108** [comment]
It's unfortunate that as the FILPlus group, you should be offering assistance from the perspective of the SPs, addressing the concerns they have. However, I've yet to see such support, which is quite disappointing.

**Comment by web3woker0001** [comment]
" The Hidde team would take this as a factual point and disseminate it widely within the community"

This is the biggest problem,  Let's take a moment to envision a world without judges to render verdicts - what would it look like? What would become of the Filecoin ecosystem? It's quite evident that the FILPlus team is reluctant to step into the shoes of a judge.

**Comment by emilytklee** [comment]
Will attach to #930 and let more people to discuss
